src/app/collection/pages/collection-ncyte/feature/collection-learning-object-card/collection-learning-object-card.component.html

```diff
@@ -0,0 +1,33 @@
+<div class = "container">
+
+    <div *ngIf = "learnObj.length !== 'micromodule'">
+        <div class="module-background">
+            <img class ="background"src="../../../../../../../assets/images/michael-dziedzic-nLFqr9Mr9H8-unsplash 1.png" alt="image not shown">
```

Comment:
You can do an aria-hidden on this image since it doesn't have any meaning for our users.

Reply:
this was from the old commit too, the latest commit doesnt have this anymore